### PR TITLE
Write the createTestFile() function in SlatwallUnitTestBase.cfc to test image files 

### PR DESCRIPTION
**Pull request by YuqingYoki**
.

---
This change is [Reviewable](https://reviewable.io/reviews/ten24/slatwall/4588)

### CODE REVIEW

File: meta/tests/unit/SlatwallUnitTestBase.cfc [hunk 1]
```diff
@@ -84,6 +85,12 @@ component extends="mxunit.framework.TestCase" output="false" {
 			flushRequired = true;
 			entityDelete( variables.persistentEntities[i] );
 		}
+		
+		for (var i = arrayLen(variables.files); i >= 1; i--) {//yuqing
+			flushRequired = true;
```
Comment (rmarchandTen24) [inline meta/tests/unit/SlatwallUnitTestBase.cfc]: you shouldn't need to set flushRequired to true. OrmFlush is used by persistedEntites and not by files

File: meta/tests/unit/SlatwallUnitTestBase.cfc [hunk 2]
```diff
@@ -106,6 +114,22 @@ component extends="mxunit.framework.TestCase" output="false" {
...
+	
+	private any function createTestFile (required string fileSourceLocalAbsolutePath, string fileDestination) {
+		var testFileDestinationPath = expandPath('Slatwall') & '/custom/TestFile';
+		request.debug(testFileDestinationPath);
+		if (arguments.fileDestination) {//yuqing
+			testFileDestinationPath = testFileDestinationPath & arguments.fileDestination;
+		}
+		
+		// Persist to the database
+		ormFlush();
```
Comment (rmarchandTen24) [inline meta/tests/unit/SlatwallUnitTestBase.cfc]: no need for an ormflush

**Comment by YuqingYoki** [comment]
I set the slatwall/asserts/images as the source of image files. Push the unit test as well to see if circleCI passes it. If it not appropriate to merge ProductTest.cfc, I will do a seperate branch just for SlatwallUnitTestBase.cfc

**Comment by YuqingYoki** [comment]
@rmarchandTen24 merged  this from my computer

**Pull request merged**